native/cocos/core/assets/EffectAsset.h

```diff
@@ -581,6 +582,8 @@ class EffectAsset final : public Asset {
     void initDefault(const ccstd::optional<ccstd::string> &uuid) override;
     bool validate() const override;
 
+private:
+    BaseEngine::EngineStatusChange::EventID _engineListener;
```

Comment:
Please rename to _engineEventID. And it is better put `private` after `protected`.

Reply:
fxied